geopandas/tools/tests/test_clip.py

```diff
@@ -353,15 +352,6 @@ def test_mixed_series(self, mixed_gdf, mask):
             and clipped.geom_type[2] == "LineString"
         )
 
-    def test_clip_warning_no_extra_geoms(self, buffered_locations, mask):
-        """Test a user warning is provided if no new geometry types are found."""
-        with pytest.warns(UserWarning):
-            clip(buffered_locations, mask, True)
-            warnings.warn(
-                "keep_geom_type was called when no extra geometry types existed.",
-                UserWarning,
-            )
-
```

Comment:
No idea what this was but `clip` never even emits such a warning so I am removing it rather than adding a `stacklevel` or `noqa` here. This test doesn't test anything.